docker-compose.yml

```diff
@@ -30,6 +30,8 @@ services:
       - LOGIN_MESSAGE=Your OTP might take awhile to get to you.
       - AWS_S3_BUCKET=file-staging.go.gov.sg
       - ROTATED_LINKS=whatsapp,passport,spsc,sppr
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
```

Comment:
Since these docker env vars do not map to any value, is there any purpose for them being here?

Reply:
They serve two purposes. First is to tell docker-compose that these two variables can be read from the terminal's environment and passed into the containers. Source [here](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers)

The second purpose is more indirect, which is to let developers know that we are expecting these two environment variables to be provided one way or another.